internal/dockercompose/fake_client.go

```diff
@@ -151,18 +149,12 @@ func (c *FakeDCClient) Project(_ context.Context, m model.DockerComposeProject)
 	}
 
 	workDir := c.WorkDir
-	projectNameOpt := func(opt *loader.Options) {
-		if workDir != "" {
-			name := filepath.Base(workDir)
-			// normalization logic from https://github.com/compose-spec/compose-go/blob/c39f6e771fe5034fe1bec40ba5f0285ec60f5efe/cli/options.go#L366-L371
-			r := regexp.MustCompile("[a-z0-9_-]")
-			name = strings.ToLower(name)
-			name = strings.Join(r.FindAllString(name, -1), "")
-			name = strings.TrimLeft(name, "_-")
-			opt.Name = name
-		} else {
-			opt.Name = "fakedc"
-		}
+	projectName := m.Name
+	if projectName == "" {
+		projectName = workDir
```

Comment:
I think you'll need to bring back the normalization logic for this case. It's possible that `strings.Lower(filepath.Base(workDir))` might be sufficient, but the test tempdirs definitely don't make it happy by default.

Alternatively, we could just always hardcode the project name to `fakedc` here - I was trying to keep behavior fairly realistic by inferring project name from the working dir, but I don't think anything test-related _actually_ cares

Reply:
Right, okay. That's easy enough to do now.

Reply:
Fixed in e757a7d718665cb7f20ccd042b3a91204debcab4